pkg/kv/repository.go

```diff
@@ -381,14 +381,14 @@ func (cer *Repository) CreateValue(value *model.ValueEntity) (*model.ValueEntity
 
 func (cer *Repository) DeleteValue(key, bucket string) error {
 	//bundle DB operations
-	dbOps := func() error {
+	dbOps := func(tx *db.TxConnection) error {
 		//delete all cache entities which were using a value of this key in this bucket
 		if err := cer.CacheDep.Invalidate().WithKey(key).WithBucket(bucket).Exec(false); err != nil {
```

Comment:
bug in code: should happen at the end of the DB calls so that in case of errors everything can be reverted